agentMET4FOF/dashboard/Dashboard.py

```diff
@@ -163,7 +163,8 @@ def is_port_available(self, ip_addr, _port):
             return True
 
 
-class AgentDashboardProcess(AgentDashboard, Process):
+
+class AgentDashboardProcess(AgentDashboard, pathos.helpers.mp.Process):
```

Comment:
I am currently looking through your changes and will finish this tomorrow morning. Can you shortly summarize what your intention was with using `pathos.helpers.mp.Process`?